python/graphstorm/config/argument.py

```diff
@@ -747,6 +748,142 @@ def eval_fanout(self):
             # By default use -1 as full neighbor
             return [-1] * self.num_layers
 
+    def _check_distill_feats(self, dt_feats):
+        """ Parse features for distillation.
+        """
+        try:
+            dt_feats_dict = {}
+            for val in dt_feats:
+                dt_feats_dict[val.split(":")[0]] = {"id_field": val.split(":")[1].split("@")[0], \
+                    "textual_field": val.split(":")[1].split("@")[1].split("/")}
+
+        except Exception:
+            assert False, f"{fot_name} distill_feats should either in format" \
+                "ntype1:ntype1_id_field@ntype1_textual_field1/ntype1_textual_field2,ntype2:...," \
+                "for example, a valid input can be 'query:query@query,asin:asin@item_name/brand'"
```

Comment:
i don't understand this format. what does it mean?

Reply:
User can specify the node types to distill. For each node type, user needs to specify the id field in textual data and the textual fields that the use wants to use for distillation.